packages/calcite-components/src/components/list/list.stories.ts

```diff
@@ -1137,3 +1150,23 @@ export const dragEnabledNestedListsIndirectChildren = (): string =>
     <div><calcite-list-item label="Depth 1" description="Item 6"></calcite-list-item></div>
     <div><calcite-list-item drag-disabled label="Depth 1" description="Item 7"></calcite-list-item></div>
   </calcite-list>`;
+
+export const nonInteractive = (): string => html`
+  <div style="width: 400px">
```

Comment:
Nitpick: is setting `width` necessary for this test?

Reply:
This is not necessary, I can take the width out.